src/SFA.DAS.Forecasting.Domain/Estimations/AccountEstimation.cs

```diff
@@ -11,17 +13,33 @@ public class AccountEstimation
     {
         internal AccountEstimationModel Model { get; }
         private readonly IVirtualApprenticeshipValidator _validator;
+        private readonly IDateTimeService _dateTimeService;
 
         public IReadOnlyCollection<VirtualApprenticeship> Apprenticeships => Model.Apprenticeships;
 
         public string Name => Model.EstimationName;
-        public bool HasValidApprenticeships => Model.Apprenticeships.Any(); //TODO: will also need to make sure all courses start after today
+        public bool HasValidApprenticeships {
+            get
+            {
+                if (!Model.Apprenticeships.Any())
+                    return false;
+
+                var latestPlannedEndDate = Model.Apprenticeships.Select(x => x.StartDate.AddMonths(x.TotalInstallments)).OrderByDescending(x => x).First().AddMonths(2).GetStartOfMonth();
+
```

Comment:
will  orderbydescending in this case brings the oldest apprenticeship

Reply:
It will bring the apprenticeship with latest end date